volumes/miovision/sql/create-mv-mio_cent.sql

```diff
@@ -0,0 +1,141 @@
+--this is a bit of a convoluded process... here are the steps I took. I'm happy to make it more efficient
+--1. wrote code to find the centreline_ids that correspond to the segments that touch miovision intersections; put these in a table
+--2. did some error checking to see if that code worked (there were 9 situations where it didn't)...
+--3. manually investigated and fixed the centrelines that needed to be fixed (in the table)
+
+--1. Here is the code that finds centreline_ids that touch miovision intersections:
+CREATE TABLE miovision_api.centerline_miovision AS (
+
+    --find all the nodes in the centreline file that touch miovision intersections. You have to match with "from" and "to" nodes to get all the segments
+    WITH cent_int AS (
+        SELECT DISTINCT
+            ter.intersection_uid,
+            ter.lat,
+            ter.lng,
+            cent.centreline_id,
+            ci.latitude,
+            ci.longitude
+        FROM miovision_api.intersections AS ter
+        LEFT JOIN prj_volume.centreline AS cent ON ter.int_id = cent.to_intersection_id
```

Comment:
I forgot why we are using this centreline, can you remind me plz :cry_cat:

Reply:
I don't know why... it may have just been a hangover from some old code. I changed it to `gis.centreline` and that actually solved a lot of the missing 🦵🏻 issues